README.rst

```diff
@@ -75,27 +75,101 @@ Awesome, isn’t it?
 
-You need to have docker installed
+Clone repository::
 
-Make sure you pull the right docker image::
+    git clone https://github.com/blue-yonder/postgraas_server
 
-    docker pull postgres:9.4
+*[optional]* Create and run a virtual environment
+
+*This step will help you to not install all needed python packages on your home computer, but in a virtual disposable environment so it doesn't cause any trouble with your local setup*::
 
-Make a virtualenv and install the requirements including the dev requirements and a local editable install
-of the package, for convenience you can install the requirements.in ::
+    python -m venv postgraas_server/
+    cd postgraas_server
+    source bin/activate
+
+Install all the project dependencies::
 
-    pip install -r requirements.in
     pip install -r requirements_dev.txt
+    pip install -r requirements_docker.in
+    pip install gunicorn
     pip install -e .
 
-For the tests you need a running postgres meta database and set some environment variables accordingly.
-There is a convenience script to set this all up using a docker postgres database::
+Docker
+-----------------
+
+Pull the right docker image::
+
+    docker pull postgres:9.4
+
+Your application.cfg file should look like this::
+
+    {
+        "metadb":
+        {
+            "db_name": "postgres",
+            "db_username": "postgres",
+            "db_pwd": "mysecret",
+            "host": "localhost",
+            "port": "5432"
+        },
+        "backend":
+        {
+            "type": "docker"
+        }
+    }
+
+Initialize a postgres DB within a docker container::
+
+    sh setup_integration_test_docker.sh
+
+Run a Docker container with the postgres image::
+
+    postgraas_init
+
+Postgres Cluster
+-----------------
+
+If you don't want to use Docker as the backend you could create a local postgres cluster
+
+Your application.cfg file should look like this::
+
+    {
+        "metadb":
+        {
+            "db_name": "postgres",
+            "db_username": "postgres",
+            "db_pwd": "mysecret",
+            "host": "localhost",
+            "port": "5432"
+        },
+        "backend":
+        {
+            "type": "pg_cluster",
+            "database": "postgres",
+            "username": "postgres",
+            "password": "mysecret",
+            "host": "localhost",
+            "port": "5432"
+        }
+    }
+
+Run postgres server::
+
+    postgres -D /usr/local/var/postgres
+
+Execute application locally
+-----------------
+
+Run the Flask application by executing this command::
+
+    python postgraas_server/postgraas_api.py
+
+After this your application should be started and you can perform GET/POST/DELETE actions to this endppoint::
 
-    . setup_integration_test_docker.sh
+    http://localhost:5000/api/v2/postgraas_instances
 
-Now you should be able to execute the tests::
+Alternatively, you can run your unit and integration testings to verify your new code::
```

Comment:
```suggestion
Alternatively, you can run your unit and integration tests to verify your new code:
```

Reply:
This is an RST file, so the double colon is actually the proper way to introduce the below code block.

Reply:
Correct, I replaced `testings` with `tests` but left the double colon

Reply:
🤦‍♂ Thanks